libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo/network/websocket/WebSocketNetworkTransport.kt

```diff
@@ -129,29 +127,6 @@ class WebSocketNetworkTransport private constructor(
       this.serverUrl = serverUrl
     }
 
-    /**
-     * Headers to add to the HTTP handshake query.
-     */
-    fun httpHeaders(headers: List<HttpHeader>) = apply {
-      this.httpHeaders = headers
-    }
-
-    /**
-     * Add a [HttpHeader] to the handshake query.
-     */
-    fun addHttpHeader(name: String, value: String) = apply {
-      this.httpHeaders = this.httpHeaders.orEmpty() + HttpHeader(name, value)
-    }
-
-    /**
-     * Add a [HttpHeader] to the handshake query.
-     */
-    @Deprecated("use addHttpHeader instead", ReplaceWith("addHttpHeader(name, value)"))
-    @ApolloDeprecatedSince(ApolloDeprecatedSince.Version.v4_0_0)
-    fun addHeader(name: String, value: String) = apply {
-      this.httpHeaders = this.httpHeaders.orEmpty() + HttpHeader(name, value)
-    }
-
```

Comment:
Another bit of precedence bites the dust. We used to have 

* ApolloClient
* ApolloRequest
* NetworkTransport

NetworkTransport is now gone for websockets. We should probably do the same for HTTP